src/scss/test-results-list.component.scss

```diff
@@ -0,0 +1,31 @@
+wh-test-results-summary-list .p-accordion .p-accordion-content {
+  background-color: transparent;
+  border: none;
+}
+
+.detail-message {
+  display: -webkit-flex;
+  display: flex;
+  -webkit-flex-direction: column;
```

Comment:
No need for vendor-specific prefixes. Our SCSS compiler should add those in for us.

Also, according to caniuse.com, the `-webkit-` prefix is no longer needed since Safari v9 which dropped 2015 (7 years ago).